diff --git a/gthread/Makefile.in b/gthread/Makefile.in
index 342f56c..bc51610 100644
--- a/gthread/Makefile.in
+++ b/gthread/Makefile.in
@@ -397,8 +397,8 @@ GTESTER_REPORT = $(top_builddir)/glib/gtester-report	# for the GLIB package
 EXTRA_DIST = makefile.msc.in gthread-posix.c gthread-win32.c \
 	gthread-none.c gthread.def gthread.rc.in
 TEST_PROGS = 
-SUBDIRS = . tests
-DIST_SUBDIRS = tests
+SUBDIRS = .
+DIST_SUBDIRS =
 AM_CPPFLAGS = \
 	$(glib_INCLUDES) 		\
 	-DG_LOG_DOMAIN=\"GThread\" 	\
