diff --git a/Makefile.in b/Makefile.in
index 272bd03..af068f9 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -86,7 +86,7 @@ REENTRANT = @REENTRANT@
 VERSION = @VERSION@
 
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = drivers include libmikmod loaders mmio playercode           docs dlapi macintosh posix os2 win32
+SUBDIRS = drivers include libmikmod loaders mmio playercode dlapi macintosh posix os2 win32
 
 
 bin_SCRIPTS = libmikmod-config
